src/dialog/dlgaboutdlg.ui

```diff
@@ -78,8 +78,27 @@
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
-        <item row="2" column="1">
-         <widget class="QLabel" name="platform_label">
+        <item row="0" column="0">
```

Comment:
I apologize for the slightly noisy diff: The markup was quite hard to read since rows/columns were in no recognizable order, therefore I've sorted them first. Reviewing commit-by-commit might be the easiest here.

Reply:
👍 
good ol' QtCreator used to randomize item order when items were added/moved in the GUI editor and diffs were unreviewable. Though, out if laziness I tried it lately and this seems to be fixed.